WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/AddressFormViewModelProtocol.swift

```diff
@@ -377,6 +378,14 @@ open class AddressFormViewModel: ObservableObject {
     func trackOnLoad() {
         // override in subclass
     }
+
+    /// Returns `true` if the fields contains a valid email or if there is no email to validate. `False` otherwise.
+    ///
+    func validateEmail() -> Bool {
+        let primaryEmailIsValid = fields.email.isEmpty || EmailFormatValidator.validate(string: fields.email)
+        let secondaryEmailIsValid = secondaryFields.email.isEmpty || EmailFormatValidator.validate(string: fields.email)
```

Comment:
This line will always be `true` as we are only using `secondaryFields` for the shipping address which does not has an email value. 

But since we are not doing that distinction at this level, I think it's safer to have the validation in case we update how we consume it in the future.